### PR TITLE
Translating more

### PR DESCRIPTION
**Pull request by batatasmor**

### CODE REVIEW

**Comment by batatasmor** [comment]
What happened?

**Comment by xlenstra** [comment]
Placeholder `[count]` not found in `Portuguese` for entry `They promised not to settle near us ([count] turns remaining)`
    Placeholder `[civilization]` not found in `Portuguese` for entry `[civilization] has started constructing [construction]!`
    Placeholder `[construction]` not found in `Portuguese` for entry `[civilization] has started constructing [construction]!`
    Placeholder `[improvement]` not found in `Portuguese` for entry `Are you sure you want to pillage this [improvement]?`
    Placeholder `[playerIndex]` not found in `Portuguese` for entry `Player [playerIndex] starting location`
    Placeholder `[resource]` not found in `Portuguese` for entry `Resource [resource] does not exist in ruleset!`
    Placeholder `[amount]` not found in `Portuguese` for entry `+[amount]% Production when constructing [param]`
    Placeholder `[amount]` not found in `Portuguese` for entry `[amount]% tile improvement construction time`
    Placeholder `[amount]` not found in `Portuguese` for entry `Nuclear weapon of Strength [amount]`
    Placeholder `[amount]` not found in `Portuguese` for entry `Blast radius [amount]`

**Comment by batatasmor** [comment]
Bruh
It returned to error.
I can't, I must be doing something wrong.
Can anyone fix it for me?

File: android/assets/jsons/translations/Portuguese.properties [hunk 1]
```diff
@@ -99,12 +97,9 @@ We will remember this. = Nós nos lembraremos disto.
...
 [civName] has denounced [targetCivName]! = [civName] condenou [targetCivName]!
- # Requires translation!
-Do you want to break your promise to [leaderName]? = 
- # Requires translation!
-We promised not to settle near them ([count] turns remaining) = 
- # Requires translation!
-They promised not to settle near us ([count] turns remaining) = 
+Do you want to break your promise to [leaderName]? = Queres quebrar a tua promessa para com [leaderName]?
+We promised not to settle near them ([count] turns remaining) = Prometemos não nos acomodarmos perto deles ([count] turnos restantes)
+They promised not to settle near us ([count] turns remaining) = Eles prometeram não se estabelecerem perto de nós ([contagem] turnos restantes)
```
Comment (sockybob) [inline android/assets/jsons/translations/Portuguese.properties:102]: For example here you translated "[count]" to "[contagem]" while you shouldn't translate it. Other errors seems to be of the same kind.

Comment (sockybob) [reply android/assets/jsons/translations/Portuguese.properties:102]: Replace those translated labels with their original english names

Comment (batatasmor) [reply android/assets/jsons/translations/Portuguese.properties:102]: ok. I will do this

**Comment by xlenstra** [comment]
```
    Placeholder `[count]` not found in `Portuguese` for entry `They promised not to settle near us ([count] turns remaining)`
    Placeholder `[playerIndex]` not found in `Portuguese` for entry `Player [playerIndex] starting location`
    Placeholder `[resource]` not found in `Portuguese` for entry `Resource [resource] does not exist in ruleset!`
```

**Comment by batatasmor** [comment]
FINALLY. Thanks to sockybob and xlenstra

**Pull request merged**